assets/src/edit-story/elements/text/output.js

```diff
@@ -151,6 +152,16 @@ export function TextOutputWithUnits({
     position: 'relative',
     padding: `${paddingStyles.vertical} ${paddingStyles.horizontal}`,
     textAlign: styles.textAlign,
+    borderRadius: `${dataToEditorY(
```

Comment:
Hm, when using the `PAGE_HEIGHT` as a reference value, does it change the original value from the output value at all?

Reply:
> Here as well -- should we add this only in case of highlight mode?

I don't see how it hurts if it's always there and this approach also simplifies complexity

> Hm, when using the `PAGE_HEIGHT` as a reference value, does it change the original value from the output value at all?

lol yea I think it just cancels out to be the input value. good catch

Reply:
> I don't see how it hurts if it's always there and this approach also simplifies complexity

Ah, right, the `textStyle` is already applied to the highlight only anyway 💡